File: 05 - Python/08-05-00-Paso-Cadenas-ULP.py
# Ejercicio Pasar una Cadena
# En Python las cadenas son inmutables
# Definir la funcion
def cambio_valor(parametro):
    parametro = 'Adios'

# Pasamos una cadena
arg_a = 'Hola'
#arg_a[0] = 'A' # Las cadenas son inmutables
print(f'Antes llamar funcion: {arg_a}')
cambio_valor(arg_a)
print(f'Despues llamar funcion: {arg_a}')

File: 04 - JavaScript/08-05-00-Paso-Cadenas-ULP.js
// Funciones en JavaScript
// Las cadenas en JavaScript son inmutables
function cambiarValor(parametro){
    parametro = 'Adios'
}

// Llamamos a la funcion
let argumento = 'Hola'
console.log(`Antes de llamar funcion: ${argumento}`)
cambiarValor(argumento)
console.log(`Despues de llamar funcion: ${argumento}`)
